.github/workflows/canary-integration-test.yml

```diff
@@ -51,16 +51,16 @@ jobs:
     - name: test external script create-external-cluster-resources.py
       run: |
         toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
-        timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr|grep -Eosq \"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\" ; do sleep 1 && echo 'waiting for the manager IP to be available'; done"
-        mgr_raw=$(kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr)
+        timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph mgr dump --name=client.rookoperator -f json|jq --raw-output .active_addr|grep -Eosq \"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\" ; do sleep 1 && echo 'waiting for the manager IP to be available'; done"
+        mgr_raw=$(kubectl -n rook-ceph exec $toolbox -- ceph mgr dump --name=client.rookoperator -f json|jq --raw-output .active_addr)
```

Comment:
Instead of passing `--name` everywhere for the toolbox, we should be able to set an env var on the toolbox so all commands will inherit by default. Try setting an env var `CEPH_ARGS="--name client.rookoperator"`. This worked for me in my minikube test cluster and allowed me to run the ceph commands.